src/components/Search/index.tsx

```diff
@@ -1,22 +1,23 @@
-import React from "react"
-import { Input, InputGroup, InputRightElement, InputGroupProps } from "@chakra-ui/react"
-import { SearchIcon } from '@chakra-ui/icons'
-
-const Search: React.FC<InputGroupProps> = ({ placeholder, ...props }) => {
-  return (
-    <InputGroup {...props}>
-      <Input 
-        h={props.h}
-        placeholder={placeholder}
-        focusBorderColor="primary.100"
-      />
-      <InputRightElement 
-        h="100%"
-        mr={1}
-        children={<SearchIcon />}
-      />
-  </InputGroup>
-  );
-};
-
-export default Search;
+import React from "react";
+import {
+  Input,
+  InputGroup,
+  InputRightElement,
+  InputGroupProps,
+} from "@chakra-ui/react";
+import { SearchIcon } from "@chakra-ui/icons";
+
+const Search: React.FC<InputGroupProps> = ({ placeholder, ...props }) => {
+  return (
+    <InputGroup {...props}>
+      <Input
+        h={props.h}
+        placeholder={placeholder}
+        focusBorderColor="primary.100"
+      />
+      <InputRightElement h="100%" mr={1} children={<SearchIcon />} />
+    </InputGroup>
+  );
+};
+
+export default Search;
```

Comment:
Please avoid default exports for components; change it to an `export const Search`.
```suggestion
export const Search: React.FC<InputGroupProps> = ({ placeholder, ...props }) => {
  return (
    <InputGroup {...props}>
      <Input
        h={props.h}
        placeholder={placeholder}
        focusBorderColor="primary.100"
      />
      <InputRightElement h="100%" mr={1} children={<SearchIcon />} />
    </InputGroup>
  );
};
```